src/app/content/components/BookBanner.tsx

```diff
@@ -87,27 +108,58 @@ const BookChapter = styled.h1`
 `;
 
 // tslint:disable-next-line:variable-name
-const BarWrapper = styled.div<{theme: Book['theme']}>`
-  position: sticky;
+export const BarWrapper = styled.div<{theme: Book['theme'], up: boolean, variant: 'mini' | undefined}>`
```

Comment:
**Musing:** Why not use `'mini' | 'expanded'` to help clarify that there are 2 different options? 

I saw that `BookChapter` is just a vanilla styled element so it's probably not important.